scripts/build_coverage.sh

```diff
@@ -169,4 +169,5 @@ lcov --capture --directory "$OUTPUT_ROOT/obj/src" --exclude="$PWD"/zzz_generated
 lcov --add-tracefile "$COVERAGE_ROOT/lcov_base.info" --add-tracefile "$COVERAGE_ROOT/lcov_test.info" --output-file "$COVERAGE_ROOT/lcov_final.info"
 genhtml "$COVERAGE_ROOT/lcov_final.info" --output-directory "$COVERAGE_ROOT/html"
 
-tar czvf "$COVERAGE_ROOT/coverage_html.tar.gz" -C "$COVERAGE_ROOT/html" .
+mv "$COVERAGE_ROOT/html" "$COVERAGE_ROOT/www"
+cp "$CHIP_ROOT/integrations/app_engine/app.yaml" "$COVERAGE_ROOT/app.yaml"
```

Comment:
Maybe add a comment about the intent here, could even be a echo saying that this thing can be deployed as a static app (does app engine have a local test command?).